File: jscomp/test/method_string_name.js
'use strict';


f = do
  "Content-Type": 3
end;

console.log(f["Content-Type"]);

function ff(x) do
  x.Hi;
  x["Content-Type"] = "hello";
  console.log((do
          "Content-Type": "hello"
        end)["Content-Type"]);
  return --[ () ]--0;
end

exports.f = f;
exports.ff = ff;
--[  Not a pure module ]--

File: jscomp/test/method_string_name.lua
__console = {log = print};


f = {
  "Content-Type" = 3
};

__console.log(f["Content-Type"]);

function ff(x) do
  x.Hi;
  x["Content-Type"] = "hello";
  __console.log(({
          "Content-Type" = "hello"
        })["Content-Type"]);
  return --[[ () ]]0;
end end

exports = {};
exports.f = f;
exports.ff = ff;
return exports;
--[[  Not a pure module ]]
